src/modules/job-post/entities/freelancer-answer.entity.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  Column,
+  Entity,
+  JoinTable,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import { JobPostQuestion } from './job-post-question.entity';
+import { Freelancer } from '../../freelancer/entities/freelancer.entity';
```

Comment:
absolute path. The same for all next files

Reply:
done